.travis.yml

```diff
@@ -10,72 +10,19 @@ env:
 
 matrix:
     include:
-        # Full (Linux, 2.7) first half
-        - os: linux
-          env: CONDA_DEPENDENCIES="numpy scipy matplotlib pandas scikit-learn h5py pillow statsmodels mayavi"
-               PIP_DEPENDENCIES="pysurfer nitime faulthandler joblib nibabel codecov nose-timer nose-faulthandler"
-               MNE_DIRS=". beamformer channels commands connectivity datasets decoding forward gui inverse_sparse io"
-        # Full (Linux, 2.7) second half
-        - os: linux
-          env: CONDA_DEPENDENCIES="numpy scipy matplotlib pandas scikit-learn h5py pillow statsmodels mayavi"
-               PIP_DEPENDENCIES="pysurfer nitime faulthandler joblib nibabel codecov nose-timer nose-faulthandler"
-               MNE_DIRS="minimum_norm preprocessing realtime simulation stats time_frequency viz"
-
-        # OSX first half
-        - os: osx
-          env: CONDA_DEPENDENCIES="numpy scipy matplotlib pandas scikit-learn h5py pillow statsmodels mayavi"
-               PIP_DEPENDENCIES="pysurfer nitime faulthandler joblib nibabel codecov nose-timer nose-faulthandler"
-               MNE_DIRS=". beamformer channels commands connectivity datasets decoding forward gui inverse_sparse io"
-        # OSX second half
-        - os: osx
-          env: CONDA_DEPENDENCIES="numpy scipy matplotlib pandas scikit-learn h5py pillow statsmodels mayavi"
-               PIP_DEPENDENCIES="pysurfer nitime faulthandler joblib nibabel codecov nose-timer nose-faulthandler"
-               MNE_DIRS="minimum_norm preprocessing realtime simulation stats time_frequency viz"
-
```

Comment:
I've removed them just to reduce the time of travis while [WIP]. I guess I should had use a more informative commit message.